examples/with-stripe-typescript/components/ElementsForm.tsx

```diff
@@ -94,21 +82,18 @@ const ElementsForm = () => {
       return
     }
 
-    // Get a reference to a mounted CardElement. Elements knows how
-    // to find your CardElement because there can only ever be one of
-    // each type of element.
-    const cardElement = elements!.getElement(CardElement)
-
     // Use your card Element with other Stripe.js APIs
-    const { error, paymentIntent } = await stripe!.confirmCardPayment(
-      response.client_secret,
-      {
-        payment_method: {
-          card: cardElement!,
-          billing_details: { name: input.cardholderName },
+    const { error } = await stripe!.confirmPayment({
+      elements,
+      confirmParams: {
+        return_url: 'http://localhost:3000/donate-with-elements',
```

Comment:
Should this be hard coded to localhost?

Reply:
It's a URL of the redirect destination.
So if we can set the `current page url`, we don't have to hard code it.
eg.)
```
return_url: window && window.location ? [location.protocol, location.hostname,'donate-with-elements'].join('/') ; 'http://localhost:3000/donate-with-elements'
```

But, it this example works well in SSG/SSR mode?
That's my worrying.